web/src/invite_user_group_picker_pill.ts

```diff
@@ -0,0 +1,39 @@
+import * as input_pill from "./input_pill";
+import {set_up_user_group} from "./pill_typeahead";
+import type {CombinedPill} from "./typeahead_helper";
+import * as user_group_pill from "./user_group_pill";
+import type {UserGroupPill} from "./user_group_pill";
+
+type SetUpPillTypeaheadConfig = {
+    pill_widget: user_group_pill.UserGroupPillWidget;
+    $pill_container: JQuery;
+};
+
+function create_item_from_group_name(
+    user_group_name: string,
+    current_items: CombinedPill[],
+): UserGroupPill | undefined {
+    return user_group_pill.create_item_from_group_name(user_group_name, current_items);
+}
+
+function set_up_pill_typeahead({pill_widget, $pill_container}: SetUpPillTypeaheadConfig): void {
+    const opts = {
+        help_on_empty_strings: true,
+        hide_on_empty_after_backspace: true,
+        invite_streams: true,
```

Comment:
Why is this `invite_streams` option set?

Reply:
Seems like a typo/substitution error; there's an `invite_user_groups` in `setup_user_groups`, but I can't tell what, if anything, that does.

Reply:
Removed. Thanks for pointing out